### PR TITLE
Spheres in The Sky

### PR DESCRIPTION
**Pull request by Daniel-Carter1**
my first map

### CODE REVIEW

File: spheres_in_the_sky/map.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+name = Spheres in The Sky
+initial_stuff = default:pick_steel,default:stick 2,default:torch 10,default:desert_cobble 70
+r = 115
+base_node = ctf_map:desert_cobble
+chests.1.n = 30
+chests.1.from = (-7.8,-45.5,-39.8)
+chests.1.to = (-22.2,-45.5,-55.2)
+chests.2.n = 30
+chests.2.from = (-4.8,-48.5,52.2)
+chests.2.to = (-19.2,-48.5,36.8)
```
Comment (JostP) [inline spheres_in_the_sky/map.conf:19]:
```suggestion
chests.1.n = 30
chests.1.from = (-8,-45,-40)
chests.1.to = (-22,-46,-55)
chests.2.n = 30
chests.2.from = (-5,-49,52)
chests.2.to = (-19,-49,37)
```
Use only integer numbers

Comment (Daniel-Carter1) [reply spheres_in_the_sky/map.conf:19]: it wont work

Comment (JostP) [reply spheres_in_the_sky/map.conf:19]: why?

Comment (Daniel-Carter1) [reply spheres_in_the_sky/map.conf:19]: i tried and it didnt

**Comment by JostP** [comment]
I am not really sure if this map is suitable for CTF. I see issues:
1. Only one(!), dig-able(!) path to enemy base = not possible to even touch flag. If they try to make new path, enemies will just snipe or shoot them off.
2. Only one small entrance into enemy base = very protected base = not even possible to touch flag.
3. Bases are too much faraway.

Let me ask discord community what they think about it.

Maybe you could solve all pointed issues with moving bases a lot closer together (distance between bases 50 blocks), making more (partially) indestructible paths and adding more entrances to the base's spheres.

**Comment by Daniel-Carter1** [comment]
> I am not really sure if this map is suitable for CTF. I see issues:
> 
> 1. Only one(!), dig-able(!) path to enemy base = not possible to even touch flag. If they try to make new path, enemies will just snipe or shoot them off.
> 2. Only one small entrance into enemy base = very protected base = not even possible to touch flag.
> 3. Bases are too much faraway.
> 
> Let me ask discord community what they think about it.
> 
> Maybe you could solve all pointed issues with moving bases a lot closer together (distance between bases 50 blocks), making more (partially) indestructible paths and adding more entrances to the base's spheres.
> 
> EDIT: Good job with custom chest zones

ohhh

**Comment by Daniel-Carter1** [comment]
what should i do?

**Comment by Daniel-Carter1** [comment]
i will try to change it

**Comment by JostP** [comment]
So you decided not to try to change it?

It is good map, but really not suitable for CTF, I hope you understand. You have map making experience and also chest zones making experience so you can create any new map if you want. Custom chest zones makes some maps very good.

**Pull request closed without merging**